spec/requests/purchases_requests_spec.rb

```diff
@@ -178,7 +178,8 @@
           purchase_params = { storage_location: new_storage_location, line_items_attributes: line_item_params }
           expect do
             put purchase_path(default_params.merge(id: purchase.id, purchase: purchase_params))
-          end.to raise_error(Errors::InsufficientAllotment)
+          end
+          # end.to raise_error(Errors::InsufficientAllotment)
```

Comment:
You shouldn't need to comment this out, you can just delete it. :) You do need tests making sure that the error is showing up correctly though. Unfortunately I don't have time to check this out in the next week or so. If you're still stuck after that, I can take a look.